action.yaml

```diff
@@ -88,6 +92,20 @@ runs:
             helm repo add --username ziltoidbot --password ${{ inputs.helm_pat }} remotecompany https://raw.githubusercontent.com/remotecompany/charts/main/index
         fi
 
+    - name: Validate Helm templates
+      shell: bash
+      run: |
+        helm secrets template mailersend helm/ --kube-version=1.19.12 --namespace "${{ inputs.validate_helm_env }}" -f helm/values."${{ inputs.validate_helm_env }}".yaml -f helm/secrets."${{ inputs.validate_helm_env }}".yaml > "${{ inputs.validate_helm_env }}".yaml 
+        /usr/local/bin/kubeval --schema-location https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/ -v 1.19.12 --strict --ignore-missing-schemas --filename=""${{ inputs.validate_helm_env }}".yaml" "${{ inputs.validate_helm_env }}".yaml
+          
+        if [ -s "${{ inputs.validate_helm_env }}".yaml ]; then
```

Comment:
no need for this dance around look at other steps

Reply:
It won't work like the other steps do, because when secrets exits with error (as it usually is the case) validation still passes because output is an empty YAML file. This workaround fixes this behaviour